### PR TITLE
DTT1 - 4.8.0 adaptation

### PR DESCRIPTION
**Pull request by fcaffieri**
# Description

---

The objective of this PR is to generate the necessary adaptations so that all the tests work with 4.8.0 packages from the Live repository as pre-release (for the tests only pre-release was used because the packages do not exist in live)

### Agent testing

<details>

<summary>Input yaml: </summary>

```
version: 0.1
description: This workflow is used to test agents deployment for DDT1 PoC
variables:
  agent-os:
    - linux-redhat-7-amd64
    - linux-redhat-7-arm64
    - linux-redhat-8-amd64
    - linux-redhat-8-arm64
    - linux-redhat-9-amd64
    - linux-redhat-9-arm64
    - linux-centos-7-amd64
    - linux-centos-7-arm64
    - linux-centos-8-amd64
    - linux-centos-8-arm64
    - linux-debian-10-amd64
    - linux-debian-10-arm64
    - linux-debian-11-amd64
    - linux-debian-11-arm64
    - linux-debian-12-amd64
    - linux-debian-12-arm64
    - linux-ubuntu-22.04-amd64
    - linux-ubuntu-22.04-arm64
    - linux-ubuntu-18.04-amd64
    - linux-ubuntu-18.04-arm64
    - linux-ubuntu-20.04-amd64
    - linux-ubuntu-20.04-arm64
    - linux-oracle-9-amd64
    - linux-amazon-2-amd64
    - linux-amazon-2-arm64
    - linux-amazon-2023-amd64
    - linux-amazon-2023-arm64

  windows-agent-os:
    - windows-desktop-10-amd64
    - windows-server-2012r2-amd64
    - windows-server-2016-amd64
    - windows-server-2019-amd64
    - windows-server-2022-amd64

  macos-agent-os:
    - macos-ventura-13-amd64
    - macos-sonoma-14-amd64

  manager-os: linux-ubuntu-22.04-amd64
  infra-provider: aws
  macos-infra-provider: vagrant
  working-dir: /tmp/dtt1-poc

tasks:
  # Unique manager allocate task
  - task: "allocate-manager-{manager-os}"
    description: "Allocate resources for the manager."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: create
          - provider: "{infra-provider}"
          - size: large
          - composite-name: "{manager-os}"
          - inventory-output: "{working-dir}/manager-{manager-os}/inventory.yaml"
          - track-output: "{working-dir}/manager-{manager-os}/track.yaml"
          - label-termination-date: "1d"
          - label-team: "qa"
          - label-issue: "https://github.com/wazuh/wazuh-qa/issues/5191"
    on-error: "abort-all"
    cleanup:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: delete
          - track-output: "{working-dir}/manager-{manager-os}/track.yaml"

  # Unique manager provision task
  - task: "provision-manager-{manager-os}"
    description: "Provision the manager."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/provision/main.py
          - inventory: "{working-dir}/manager-{manager-os}/inventory.yaml"
          - install:
            - component: wazuh-manager
              type: assistant
              version: 4.8.0
              live: False
    depends-on:
      - "allocate-manager-{manager-os}"
    on-error: "abort-all"


  # Unique agent allocate task
  - task: "allocate-agent-{agent}"
    description: "Allocate resources for the agent."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: create
          - provider: "{infra-provider}"
          - size: small
          - composite-name: "{agent}"
          - inventory-output: "{working-dir}/agent-{agent}/inventory.yaml"
          - track-output: "{working-dir}/agent-{agent}/track.yaml"
          - label-termination-date: "1d"
          - label-team: "qa"
          - label-issue: "https://github.com/wazuh/wazuh-qa/issues/5191"
    on-error: "abort-all"
    foreach:
      - variable: agent-os
        as: agent
    cleanup:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: delete
          - track-output: "{working-dir}/agent-{agent}/track.yaml"


  # Generic agent test task
  - task: "run-agent-{agent}-tests"
    description: "Run tests install for the agent {agent}."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/testing/main.py
          - targets:
            - wazuh-1: "{working-dir}/manager-{manager-os}/inventory.yaml"
            - agent: "{working-dir}/agent-{agent}/inventory.yaml"
          - tests: "install,registration,connection,basic_info,restart,stop,uninstall"
          - component: "agent"
          - wazuh-version: "4.8.0"
          - wazuh-revision: "40811"
          - live: False
    foreach:
      - variable: agent-os
        as: agent
    depends-on:
      - "allocate-agent-{agent}"
      - "provision-manager-{manager-os}"



  # Unique agent allocate task
  - task: "allocate-macos-agent-{agent}"
    description: "Allocate resources for the agent."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: create
          - provider: "{macos-infra-provider}"
          - size: small
          - composite-name: "{agent}"
          - inventory-output: "{working-dir}/agent-{agent}/inventory.yaml"
          - track-output: "{working-dir}/agent-{agent}/track.yaml"
          - label-termination-date: "1d"
          - label-team: "qa"
          - label-issue: "https://github.com/wazuh/wazuh-qa/issues/5191"
    on-error: "abort-all"
    foreach:
      - variable: macos-agent-os
        as: agent
    cleanup:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: delete
          - track-output: "{working-dir}/agent-{agent}/track.yaml"


  # Generic agent test task
  - task: "run-macos-agent-{agent}-tests"
    description: "Run tests install for the agent {agent}."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/testing/main.py
          - targets:
            - wazuh-1: "{working-dir}/manager-{manager-os}/inventory.yaml"
            - agent: "{working-dir}/agent-{agent}/inventory.yaml"
          - tests: "install,registration,connection,basic_info,restart,stop,uninstall"
          - component: "agent"
          - wazuh-version: "4.8.0"
          - wazuh-revision: "40811"
          - live: False
    foreach:
      - variable: macos-agent-os
        as: agent
    depends-on:
      - "allocate-macos-agent-{agent}"
      - "provision-manager-{manager-os}"


  # Unique agent allocate task
  - task: "allocate-windows-agent-{agent}"
    description: "Allocate resources for the agent."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: create
          - provider: "{infra-provider}"
          - size: small
          - composite-name: "{agent}"
          - inventory-output: "{working-dir}/agent-{agent}/inventory.yaml"
          - track-output: "{working-dir}/agent-{agent}/track.yaml"
          - label-termination-date: "1d"
          - label-team: "qa"
          - label-issue: "https://github.com/wazuh/wazuh-qa/issues/5191"
    on-error: "abort-all"
    foreach:
      - variable: windows-agent-os
        as: agent
    cleanup:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: delete
          - track-output: "{working-dir}/agent-{agent}/track.yaml"


  # Generic agent test task
  - task: "run-windows-agent-{agent}-tests"
    description: "Run tests install for the agent {agent}."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/testing/main.py
          - targets:
            - wazuh-1: "{working-dir}/manager-{manager-os}/inventory.yaml"
            - agent: "{working-dir}/agent-{agent}/inventory.yaml"
          - tests: "install,registration,connection,basic_info,restart,stop,uninstall"
          - component: "agent"
          - wazuh-version: "4.8.0"
          - wazuh-revision: "40811"
          - live: False
    foreach:
      - variable: windows-agent-os
        as: agent
    depends-on:
      - "allocate-windows-agent-{agent}"
      - "provision-manager-{manager-os}"

```
</details>

Full Log: 

[Test-agents-complete.log](https://github.com/user-attachments/files/15551867/Test-agents-complete.log)

Summary Result: 

| Agent | Test result |
| :- | :-: |
| linux-redhat-7-amd64 | :green_circle: | 
| linux-redhat-7-arm64 | :green_circle: | 
| linux-redhat-8-amd64 | :green_circle: | 
| linux-redhat-8-arm64 | :green_circle: | 
| linux-redhat-9-amd64 | :green_circle: | 
| linux-redhat-9-arm64 | :green_circle: | 
| linux-centos-7-amd64 | :green_circle: | 
| linux-centos-7-arm64 | :green_circle: | 
| linux-centos-8-amd64 | :green_circle: | 
| linux-centos-8-arm64 | :green_circle: | 
| linux-debian-10-amd64 | :green_circle: | 
| linux-debian-10-arm64 | :green_circle: | 
| linux-debian-11-amd64 | :green_circle: | 
| linux-debian-11-arm64 | :green_circle: | 
| linux-debian-12-amd64 | :green_circle: | 
| linux-debian-12-arm64 | :green_circle: | 
| linux-ubuntu-22.04-amd64 | :green_circle: | 
| linux-ubuntu-22.04-arm64 | :green_circle: | 
| linux-ubuntu-18.04-amd64 | :green_circle: | 
| linux-ubuntu-18.04-arm64 | :green_circle: | 
| linux-ubuntu-20.04-amd64 | :green_circle: | 
| linux-ubuntu-20.04-arm64 | :green_circle: | 
| linux-oracle-9-amd64 | :green_circle: | 
| linux-amazon-2-amd64 | :green_circle: | 
| linux-amazon-2-arm64 | :green_circle: | 
| linux-amazon-2023-amd64 | :green_circle: | 
| linux-amazon-2023-arm64  | :green_circle: | 
| windows-server-2012r2-amd64 | :green_circle: | 
| windows-server-2016-amd64 | :green_circle: | 
| windows-server-2019-amd64 | :green_circle: | 
| macos-ventura-13-amd64 | :green_circle: | 
| macos-sonoma-14-amd64 | :green_circle: | 

Test of remaining Windows agents:

| Agent | Test result |
| :- | :-: | 
| windows-server-2022-amd64 | :green_circle: |
| windows-desktop-10-amd64 | :green_circle: | 

<details>

<summary>Input yaml: </summary>

```
version: 0.1
description: This workflow is used to test agents deployment for DDT1 PoC
variables:
  windows-agent-os:
    - windows-desktop-10-amd64
    - windows-server-2022-amd64

  manager-os: linux-ubuntu-22.04-amd64
  infra-provider: aws
  macos-infra-provider: vagrant
  working-dir: /tmp/dtt1-poc

tasks:
  # Unique manager allocate task
  - task: "allocate-manager-{manager-os}"
    description: "Allocate resources for the manager."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: create
          - provider: "{infra-provider}"
          - size: large
          - composite-name: "{manager-os}"
          - inventory-output: "{working-dir}/manager-{manager-os}/inventory.yaml"
          - track-output: "{working-dir}/manager-{manager-os}/track.yaml"
          - label-termination-date: "1d"
          - label-team: "qa"
          - label-issue: "https://github.com/wazuh/wazuh-qa/issues/5191"
    on-error: "abort-all"
    cleanup:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: delete
          - track-output: "{working-dir}/manager-{manager-os}/track.yaml"

  # Unique manager provision task
  - task: "provision-manager-{manager-os}"
    description: "Provision the manager."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/provision/main.py
          - inventory: "{working-dir}/manager-{manager-os}/inventory.yaml"
          - install:
            - component: wazuh-manager
              type: assistant
              version: 4.8.0
              live: False
    depends-on:
      - "allocate-manager-{manager-os}"
    on-error: "abort-all"


   #Unique agent allocate task
  - task: "allocate-windows-agent-{agent}"
    description: "Allocate resources for the agent."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: create
          - provider: "{infra-provider}"
          - size: large
          - composite-name: "{agent}"
          - inventory-output: "{working-dir}/agent-{agent}/inventory.yaml"
          - track-output: "{working-dir}/agent-{agent}/track.yaml"
          - label-termination-date: "1d"
          - label-team: "qa"
          - label-issue: "https://github.com/wazuh/wazuh-qa/issues/5191"
    on-error: "abort-all"
    foreach:
      - variable: windows-agent-os
        as: agent
    cleanup:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: delete
          - track-output: "{working-dir}/agent-{agent}/track.yaml"


  # Generic agent test task
  - task: "run-windows-agent-{agent}-tests"
    description: "Run tests install for the agent {agent}."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/testing/main.py
          - targets:
            - wazuh-1: "{working-dir}/manager-{manager-os}/inventory.yaml"
            - agent: "{working-dir}/agent-{agent}/inventory.yaml"
          - tests: "install,registration,connection,basic_info,restart,stop,uninstall"
          - component: "agent"
          - wazuh-version: "4.8.0"
          - wazuh-revision: "40812"
          - live: False
    foreach:
      - variable: windows-agent-os
        as: agent
    depends-on:
      - "allocate-windows-agent-{agent}"
      - "provision-manager-{manager-os}"

```
</details>

Logs:

[Test-agents-complete_only_failures.log](https://github.com/user-attachments/files/15555540/Test-agents-complete_only_failures.log)

Test remaining MacOS agents:

| Agent | Test result |
| :- | :-: |
| macos-ventura-13-arm64 | :green_circle: |
| macos-sonoma-14-arm64 | :green_circle: | 

<details>

<summary>Input yaml: </summary>

```
version: 0.1
description: This workflow is used to test agents deployment for DDT1 PoC
variables:

  macos-agent-os:
    - macos-ventura-13-arm64
    - macos-sonoma-14-arm64

  manager-os: linux-ubuntu-22.04-amd64
  infra-provider: aws
  macos-infra-provider: vagrant
  working-dir: /tmp/dtt1-poc

tasks:
  # Unique manager allocate task
  - task: "allocate-manager-{manager-os}"
    description: "Allocate resources for the manager."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: create
          - provider: "{infra-provider}"
          - size: large
          - composite-name: "{manager-os}"
          - inventory-output: "{working-dir}/manager-{manager-os}/inventory.yaml"
          - track-output: "{working-dir}/manager-{manager-os}/track.yaml"
          - label-termination-date: "1d"
          - label-team: "qa"
          - label-issue: "https://github.com/wazuh/wazuh-qa/issues/5191"
    on-error: "abort-all"
    cleanup:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: delete
          - track-output: "{working-dir}/manager-{manager-os}/track.yaml"

  # Unique manager provision task
  - task: "provision-manager-{manager-os}"
    description: "Provision the manager."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/provision/main.py
          - inventory: "{working-dir}/manager-{manager-os}/inventory.yaml"
          - install:
            - component: wazuh-manager
              type: assistant
              version: 4.8.0
              live: False
    depends-on:
      - "allocate-manager-{manager-os}"
    on-error: "abort-all"


  # Unique agent allocate task
  - task: "allocate-macos-agent-{agent}"
    description: "Allocate resources for the agent."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: create
          - provider: "{macos-infra-provider}"
          - size: large
          - composite-name: "{agent}"
          - inventory-output: "{working-dir}/agent-{agent}/inventory.yaml"
          - track-output: "{working-dir}/agent-{agent}/track.yaml"
          - label-termination-date: "1d"
          - label-team: "qa"
          - label-issue: "https://github.com/wazuh/wazuh-qa/issues/5191"
    on-error: "abort-all"
    foreach:
      - variable: macos-agent-os
        as: agent
    cleanup:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: delete
          - track-output: "{working-dir}/agent-{agent}/track.yaml"


  # Generic agent test task
  - task: "run-macos-agent-{agent}-tests"
    description: "Run tests install for the agent {agent}."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/testing/main.py
          - targets:
            - wazuh-1: "{working-dir}/manager-{manager-os}/inventory.yaml"
            - agent: "{working-dir}/agent-{agent}/inventory.yaml"
          - tests: "install,registration,connection,basic_info,restart,stop,uninstall"
          - component: "agent"
          - wazuh-version: "4.8.0"
          - wazuh-revision: "40812"
          - live: False
    foreach:
      - variable: macos-agent-os
        as: agent
    depends-on:
      - "allocate-macos-agent-{agent}"
      - "provision-manager-{manager-os}"


```

</details>

Logs:

[Test-agents-macos_arm64.log](https://github.com/user-attachments/files/15568259/Test-agents-macos_arm64.log)

---

### Manager testing:

<details>

<summary>Input yaml: </summary>

```
version: 0.1
description: This workflow is used to test manager deployment for DDT1 PoC
variables:
  manager-os:
    - linux-ubuntu-20.04-amd64
    - linux-ubuntu-22.04-amd64
    - linux-amazon-2-amd64
    - linux-redhat-7-amd64
    - linux-redhat-8-amd64
    - linux-redhat-9-amd64
    - linux-centos-7-amd64
    - linux-centos-8-amd64
    - linux-debian-10-amd64
    - linux-debian-11-amd64
    - linux-debian-12-amd64
  infra-provider: aws
  working-dir: /tmp/dtt1-poc

tasks:
  # Unique manager allocate task
  - task: "allocate-manager-{manager}"
    description: "Allocate resources for the manager."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: create
          - provider: "{infra-provider}"
          - size: large
          - composite-name: "{manager}"
          - inventory-output: "{working-dir}/manager-{manager}/inventory.yaml"
          - track-output: "{working-dir}/manager-{manager}/track.yaml"
          - label-termination-date: "1d"
          - label-team: "qa"
    on-error: "abort-all"
    foreach:
      - variable: manager-os
        as: manager
    cleanup:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: delete
          - track-output: "{working-dir}/manager-{manager}/track.yaml"

  # Generic manager test task
  - task: "run-manager-tests"
    description: "Run tests install for the manager."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/testing/main.py
          - targets:
            - wazuh-1: "{working-dir}/manager-linux-centos-7-amd64/inventory.yaml"
            - wazuh-2: "{working-dir}/manager-linux-ubuntu-20.04-amd64/inventory.yaml"
            - wazuh-3: "{working-dir}/manager-linux-ubuntu-22.04-amd64/inventory.yaml"
            - wazuh-4: "{working-dir}/manager-linux-redhat-7-amd64/inventory.yaml"
            - wazuh-5: "{working-dir}/manager-linux-redhat-8-amd64/inventory.yaml"
            - wazuh-6: "{working-dir}/manager-linux-redhat-9-amd64/inventory.yaml"
            - wazuh-7: "{working-dir}/manager-linux-centos-8-amd64/inventory.yaml"
            - wazuh-8: "{working-dir}/manager-linux-debian-10-amd64/inventory.yaml"
            - wazuh-9: "{working-dir}/manager-linux-debian-11-amd64/inventory.yaml"
            - wazuh-10: "{working-dir}/manager-linux-debian-12-amd64/inventory.yaml"
            - wazuh-11: "{working-dir}/manager-linux-amazon-2-amd64/inventory.yaml"
          - tests: "install,restart,stop,uninstall"
          - component: "manager"
          - wazuh-version: "4.8.0"
          - wazuh-revision: "40812"
          - live: False
    depends-on:
      - "allocate-manager-linux-ubuntu-20.04-amd64"
      - "allocate-manager-linux-ubuntu-22.04-amd64"
      - "allocate-manager-linux-amazon-2-amd64"
      - "allocate-manager-linux-redhat-7-amd64"
      - "allocate-manager-linux-redhat-8-amd64"
      - "allocate-manager-linux-redhat-9-amd64"
      - "allocate-manager-linux-centos-7-amd64"
      - "allocate-manager-linux-centos-8-amd64"
      - "allocate-manager-linux-debian-10-amd64"
      - "allocate-manager-linux-debian-11-amd64"
      - "allocate-manager-linux-debian-12-amd64"




```

</details>

Full Log: 

[Test-manager-complete.log](https://github.com/user-attachments/files/15744364/Test-manager-complete.log)

Summury Result: 

| Agent | Test result | 
| :- | :-: |
| linux-ubuntu-20.04-amd6 | :green_circle: | 
| linux-ubuntu-22.04-amd6 | :green_circle: | 
| linux-redhat-7-amd64 | :green_circle: | 
| linux-redhat-8-amd64 | :green_circle: | 
| linux-redhat-9-amd64 | :green_circle: | 
| linux-amazon-2-amd64 | :green_circle: | 
| linux-centos-8-amd64 | :green_circle: | 
| linux-centos-7-amd64 | :green_circle: | 
| linux-debian-10-amd64 | :green_circle: | 
| linux-debian-11-amd64 | :green_circle: | 
| linux-debian-12-amd64 | :green_circle: | 

---

### Test summary:

| Agent | Test result | 
| :- | :-: | 
| linux-ubuntu-20.04-amd64 | :green_circle: | 
| linux-ubuntu-22.04-amd64 | :green_circle: | 
| linux-amazon-2-amd64 | :green_circle: | 
| linux-redhat-7-amd64 | :green_circle: | 
| linux-redhat-8-amd64 | :green_circle: | 
| linux-redhat-9-amd64 | :green_circle: | 
| linux-centos-7-amd64 | :green_circle: | 
| linux-centos-8-amd64 | :green_circle: | 
| linux-debian-10-amd64 | :green_circle: | 
| linux-debian-11-amd64 | :green_circle: | 
| linux-debian-12-amd64 | :green_circle: | 

<details>

<summary>Input yaml: </summary>

```
version: 0.1
description: This workflow is used to test the Wazuh manager deployment for DDT1 PoC
variables:
  central_components-os:
    - linux-ubuntu-20.04-amd64
    - linux-ubuntu-22.04-amd64
    - linux-amazon-2-amd64
    - linux-redhat-7-amd64
    - linux-redhat-8-amd64
    - linux-redhat-9-amd64
    - linux-centos-7-amd64
    - linux-centos-8-amd64
    - linux-debian-10-amd64
    - linux-debian-11-amd64
    - linux-debian-12-amd64
  infra-provider: aws
  working-dir: /tmp/dtt1-poc

tasks:
  # Unique central components allocate task
  - task: "allocate-central_components-{central_components}"
    description: "Allocate resources for the central_components."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: create
          - provider: "{infra-provider}"
          - size: large
          - composite-name: "{central_components}"
          - inventory-output: "{working-dir}/central_components-{central_components}/inventory.yaml"
          - track-output: "{working-dir}/central_components-{central_components}/track.yaml"
          - label-termination-date: "1d"
          - label-team: "qa"
    on-error: "abort-all"
    foreach:
      - variable: central_components-os
        as: central_components
    cleanup:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: delete
          - track-output: "{working-dir}/central_components-{central_components-os}/track.yaml"

  # Generic manager test task
  - task: "run-central_components-{central_components}-tests"
    description: "Run tests install for the central_components."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/testing/main.py
          - targets:
            - wazuh-1: "{working-dir}/central_components-{central_components}/inventory.yaml"
          - tests: "install,restart,stop,uninstall"
          - component: "central_components"
          - wazuh-version: "4.8.0"
          - wazuh-revision: "40812"
          - live: False
    on-error: "abort-all"
    foreach:
      - variable: central_components-os
        as: central_components

```

</details>

Full Log: 

[Test-CC-complete.log](https://github.com/user-attachments/files/15744619/Test-CC-complete.log)

---

Connection test was implemented between the Wazuh manager and indexer

| Agent | Test result | 
| :- | :-: | 
| linux-ubuntu-20.04-amd64 | :green_circle: | 
| linux-ubuntu-22.04-amd64 | :green_circle: | 
| linux-amazon-2-amd64 | :green_circle: | 
| linux-redhat-7-amd64 | :green_circle: | 
| linux-redhat-8-amd64 | :green_circle: | 
| linux-redhat-9-amd64 | :green_circle: | 
| linux-centos-7-amd64 | :green_circle: | 
| linux-centos-8-amd64 | :green_circle: | 
| linux-debian-10-amd64 | :green_circle: | 
| linux-debian-11-amd64 | :green_circle: | 
| linux-debian-12-amd64 | :green_circle: | 

<details>

<summary>Input yaml: </summary>

```
version: 0.1
description: This workflow is used to test the Wazuh manager deployment for DDT1 PoC
variables:
  central_components-os:
    - linux-ubuntu-20.04-amd64
    - linux-ubuntu-22.04-amd64
    - linux-amazon-2-amd64
    - linux-redhat-7-amd64
    - linux-redhat-8-amd64
    - linux-redhat-9-amd64
    - linux-centos-7-amd64
    - linux-centos-8-amd64
    - linux-debian-10-amd64
    - linux-debian-11-amd64
    - linux-debian-12-amd64
  infra-provider: aws
  working-dir: /tmp/dtt1-poc

tasks:
  # Unique central components allocate task
  - task: "allocate-central_components-{central_components}"
    description: "Allocate resources for the central_components."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: create
          - provider: "{infra-provider}"
          - size: large
          - composite-name: "{central_components}"
          - inventory-output: "{working-dir}/central_components-{central_components}/inventory.yaml"
          - track-output: "{working-dir}/central_components-{central_components}/track.yaml"
          - label-termination-date: "1d"
          - label-team: "qa"
    on-error: "abort-all"
    foreach:
      - variable: central_components-os
        as: central_components
    cleanup:
      this: process
      with:
        path: python3
        args:
          - modules/allocation/main.py
          - action: delete
          - track-output: "{working-dir}/central_components-{central_components-os}/track.yaml"

  # Generic manager test task
  - task: "run-central_components-{central_components}-tests"
    description: "Run tests install for the central_components."
    do:
      this: process
      with:
        path: python3
        args:
          - modules/testing/main.py
          - targets:
            - wazuh-1: "{working-dir}/central_components-{central_components}/inventory.yaml"
          - tests: "install,restart,stop,uninstall"
          - component: "central_components"
          - wazuh-version: "4.8.0"
          - wazuh-revision: "40812"
          - live: False
    on-error: "abort-all"
    foreach:
      - variable: central_components-os
        as: central_components

```

</details>

Full Log: 

[Test-CC-complete.log](https://github.com/user-attachments/files/15787990/Test-CC-complete.log)

### CODE REVIEW

File: deployability/modules/testing/tests/helpers/manager.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import requests
 import time
 
-from .constants import CLUSTER_CONTROL, AGENT_CONTROL, WAZUH_CONF, WAZUH_ROOT
+from .constants import CLUSTER_CONTROL, AGENT_CONTROL, WAZUH_CONF, WAZUH_ROOT, WAZUH_LOG
```
Comment (pro-akim) [inline deployability/modules/testing/tests/helpers/manager.py:8]: I am not finding changes in constants.py where WAZUH_LOG should be seen.

**Review by pro-akim — CHANGES_REQUESTED** [review]
## Review notes
LGTM

**Review by pro-akim — APPROVED** [review]
## Review Notes
LGTM

**Review by rauldpm — APPROVED** [review]
LGTM

**Pull request merged**